src/misc/blog-posts.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
It's ok for now, but at some point we should write a script to automatically put together this data when a new article is added.

Reply:
yeah i was looking into that...i wanted to implement a script which runs through whole articles folder and automatically imports everything...